src/trapfunc.cpp

```diff
@@ -627,9 +627,6 @@ void trapfunc::dissector( Creature *c, const tripoint &p )
             n->deal_damage( nullptr, bp_foot_r, damage_instance( DT_CUT, 10 ) );
         } else if( z != nullptr ) {
             z->apply_damage( nullptr, bp_torso, 60 );
-            if( z->is_dead() ) {
-                z->explode();
```

Comment:
But it also makes the corpses vanish, which makes the broken cyborg (which spawns nearby and usually runs into a dissector before the player encounters them) useless. The player will only encounter a bloody dissector and nothing more. We may as well spawn a standard zombie there.